CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Support mod-login-saml's requirement for the new authtoken interface in addition to the old interface. Refs STCOR-76. Available from v2.12.1.
 * Expose timeZone through react-intl provider
 * Export classes and functions intended for external use
+* Extract LastVisited to a separate module. Fixes STCOR-254. Available from v2.12.2.
```

Comment:
This is picky, but I would not characterise this as a separate _module_ -- a term that has a specific meaning in the world of JavaScript (and indeed another in the world of Stripes).

Reply:
Good point @MikeTaylor! I will change it now.